packages/models/user_profiles.coffee

```diff
@@ -1,4 +1,11 @@
 UserProfiles = new Mongo.Collection('userProfile')
+UserProfiles.allow
+  insert: (userId, doc)  ->
+    doc.userId == userId
+  update: (userId, doc, fields, modifier) ->
+    doc.userId == userId
+  remove: (userId, doc) ->
+    doc.userId == userId
```

Comment:
I think we're only modifying the profile from the server, so these don't do anything.